### PR TITLE
Missing translated terms

### PR DESCRIPTION
**Pull request by adammeyer**

### CODE REVIEW

**Comment by adammeyer** [comment]
@soberstadt look good?

File: app/views/contacts/index.html.erb [hunk 1]
```diff
@@ -183,10 +183,10 @@
       <tr>
         <td colspan="10">
           <div style="float:left">
-            Displaying <b>{{page.from}}&nbsp;-&nbsp;{{page.to}}</b> of <b>{{totalContacts}}</b>
+            <%= _('Displaying') %> <b>{{page.from}}&nbsp;-&nbsp;{{page.to}}</b> of <b>{{totalContacts}}</b>
```
Comment (soberstadt) [inline app/views/contacts/index.html.erb]: 'of' should be translated here. I don't know if there is a better way to do this or if some translations might have different grammar for 'Displaying x of y'. @twinge thoughts?

File: app/views/contacts/index.html.erb [hunk 2]
```diff
@@ -208,7 +208,11 @@
     </table>
     <div class="contacts-scrollable" id="contacts-scrollable">
       <table class="tablelist" ng-class="{'contact_loading': contactsLoading}">
-        <tr ng-show="contacts.length === 0 && contactQuery.viewPrefsLoaded"><td style="color:red;">Sorry, there are no contacts that match the filters and/or tags you have selected. Try <a href="/contacts/new">adding a new contact</a> or <a href="#" ng-click="resetFilters()">resetting your filters</a>.</td></tr>
+        <tr ng-show="contacts.length === 0 && contactQuery.viewPrefsLoaded">
+          <td style="color:red;">
+            Sorry, there are no contacts that match the filters and/or tags you have selected. Try <a href="/contacts/new">adding a new contact</a> or <a href="#" ng-click="resetFilters()">resetting your filters</a>.
```
Comment (soberstadt) [inline app/views/contacts/index.html.erb]: You might have missed this one or its hard with the link. I don't know much about advanced i18n

**Comment by soberstadt** [comment]
Good thinking on this! Other than those two harder things I pointed out, it looks good. I know we aren't shooting for 100% yet, so you could merge now if you want.

**Pull request merged**